zookeeper-server/src/main/java/org/apache/zookeeper/server/FinalRequestProcessor.java

```diff
@@ -95,10 +91,14 @@
 public class FinalRequestProcessor implements RequestProcessor {
     private static final Logger LOG = LoggerFactory.getLogger(FinalRequestProcessor.class);
 
+    public static final String PATH_MAX_DEPTH = "zookeeper.path.maxDepth";
```

Comment:
I can't find this constant PATH_MAX_DEPTH used in this patch , it looks like PATH_STATS_MAX_DEPTH

Reply:
good catch, it's dead variable and I removed it.